core/domain/state_domain.py

```diff
@@ -2827,6 +3026,17 @@ def to_dict(self):
         }
 
 
+AcceptableConversionFnType = Union[
+    Callable[[SubtitledHtml, str], SubtitledHtml],
+    Callable[
+        [Union[SubtitledHtml, SubtitledUnicode], str],
+        Union[SubtitledHtmlDict, SubtitledUnicodeDict]
+    ],
+    Callable[[Dict[str, str], str], Union[SubtitledHtml, SubtitledUnicode]],
+    Callable[[SubtitledHtml, str], List[str]]
+]
```

Comment:
Should this be placed at the top of this file?

Reply:
I think No, If we do this then at runtime it will throw an error because classes `SubtitledHtml`, `SubtitledHtmlDict` and etc are defined below this!

if we import it under the `if MYPY` clause then everything will work fine but IDE throws an error by putting yellow ( on my VS code ) curly lines under the words.